deps/wazuh_testing/wazuh_testing/tools/services.py

```diff
@@ -89,14 +89,18 @@ def control_service(action, daemon=None, debug_mode=False):
                 command = subprocess.run(["net", action, "WazuhSvc"], stderr=subprocess.PIPE)
                 result = command.returncode
                 if result != 0:
-                    if action == 'stop' and 'The Wazuh service is not started.' in command.stderr.decode():
+                    error = command.stderr.decode()
+                    if 'The service is starting or stopping' in error:
+                        time.sleep(1)
+                        continue
```

Comment:
I suspect that the idea of this change is to wait one second until the command settles down, but in the next iteration what's the output of running "stop" if the service was already stopped? in case is non-zero the test will fail.

Reply:
nvm, this case will be handled here https://github.com/wazuh/wazuh-qa/pull/4741/files#diff-d44858bebedf9fe159d885c1f40ff217b89ec7abe92ea920fccb7c2319fc5fe1R96